### PR TITLE
add documentation for zengge-wifi

### PR DESCRIPTION
**Pull request by autinerd**
**Description:**
Add documentation for new platform 'zengge-wifi'

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#19568

## Checklist:

- [X] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [ ] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/light.zengge-wifi.markdown [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+To enable the lights, add the following lines to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+light:
+  - platform: zengge-wifi
+    devices:
+      "192.168.1.39":
+        name: Living Room
```
Comment (frenck) [inline source/_components/light.zengge-wifi.markdown]: We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.

File: source/_components/light.zengge-wifi.markdown [hunk 2]
```diff
@@ -0,0 +1,46 @@
+---
+layout: page
+title: "Zengge Wi-Fi"
+description: "Instructions on how to integrate Zengge Wi-Fi bulbs into Home Assistant."
+date: 2017-01-14 08:00
```
Comment (frenck) [inline source/_components/light.zengge-wifi.markdown]: This date seems off...?

**Review by frenck — CHANGES_REQUESTED** [review]
@autinerd Thanks for the PR. Left you 2 small comments. Could you take a look? 👍

**Comment by autinerd** [comment]
I have now changed the configuration scheme, so it's simpler. The date is also corrected.

**Comment by autinerd** [comment]
In home-assistant/home-assistant#19568 it was found out that this platform already exists, so this can be closed as well.

**Pull request closed without merging**